packages/react-router/docs/guides/blocked-updates.md

```diff
@@ -1,6 +1,6 @@
 # Dealing with Update Blocking
 
-React's `shouldComponentUpdate` lifecycle method optimizes rendering performance by blocking the re-rendering of components. Unfortunately, this means that React Router's location-aware components can become out of sync with the current location if their re-rendering was prevented. For example, a `<NavLink>` that is "active" for a location may continue to be styled as active after a location change, despite no longer matching the current URL, because it is not updated when the location changed.
+React's `PureComponent` and `shouldComponentUpdate` lifecycle method optimize rendering performance by blocking the re-rendering of components. Unfortunately, this means that React Router's location-aware components can become out of sync with the current location if their re-rendering was prevented. For example, a `<NavLink>` that is "active" for a location may continue to be styled as active after a location change, despite no longer matching the current URL, because it is not updated when the location changed.
```

Comment:
> React's `PureComponent` and _the_ `shouldComponentUpdate` lifecycle method

Otherwise, it sounds like you're referencing the `PureComponent` lifecycle method.